rpms/awips2.upc/Installer.ldm/patch/etc/pqact.main

```diff
@@ -697,6 +697,12 @@ HDS	^LDIZ1[1-7] KWNS (..)(..)(..)
 HDS	^LDIZ48 KWNS (..)(..)(..)
 	FILE	-edex -log -close
 	/awips2/data_store/spc/(\1:yyyy)(\1:mm)(\1:dd)(\1:hh).day4_outlook.grib2
+# Prob of Severe Thunderstorm Grids
```

Comment:
Where did this come from? Something that was in v18 previously, or somewhere else?

Reply:
What I had added into the pqact.conf.priority that had the HSD ANY error, I had forgot to copy it to this main pqact